docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Test
```

Comment:
Remove this file?

Reply:
It's useful for locally testing the builder, I'll keep it but adjust it